### PR TITLE
Add US endpoint

### PR DESCRIPTION
**Pull request by TomEros**
Add the us endpoint

### CODE REVIEW

File: ovh/client.py [hunk 1]
```diff
@@ -74,6 +74,7 @@
 #: Mapping between OVH API region names and corresponding endpoints
 ENDPOINTS = {
     'ovh-eu': 'https://eu.api.ovh.com/1.0',
+    'ovh-us': 'https://api.ovh.us/1.0',
```
Comment (Rbeuque74) [inline ovh/client.py]: URL is incorrect. Official URL for OVH US API is https://api.ovhcloud.com

**Comment by geoffreybauduin** [comment]
Can you please sign your commits, as per the contribution guidelines?

File: README.rst [hunk 2]
```diff
@@ -545,6 +547,16 @@ OVH Europe
 - **Create application credentials**: https://eu.api.ovh.com/createApp/
 - **Create script credentials** (all keys at once): https://eu.api.ovh.com/createToken/
 
+OVH US
+----------
+
+
+- **Documentation**: https://api.ovhcloud.com/
+- **Community support**: api-subscribe@ml.ovh.net
```
Comment (Alkorin) [inline README.rst]: For US too?

Comment (rbeuque74) [reply README.rst]: Indeed, this have to be removed

**Pull request merged**